bin/docker-helper.sh

```diff
@@ -50,7 +49,15 @@ function _fail {
 }
 
 function _get_current_version() {
-    cat ${VERSION_FILE}
+    # setuptools_scm will be installed transparently if not available, Python3 is expected to be present
+    if ! python3 -m pip -qqq list | grep -F "setuptools_scm"; then
+      python3 -m pip install -qqq setuptools_scm > /dev/null 2>&1
+    fi
```

Comment:
I understand the reasoning for this, but I would like to discuss if this is something we can somehow avoid? Especially since developers might use `make docker-build` and use this script, and might not have the venv activated - it might lead to some modifications to the global / pyenv install users did not assume.

Same for the `release-helper.sh`

Reply:
I think we can avoid this specific issue by depending on the `venv` make target in the `docker-build` make targets

/cc @alexrashed

Reply:
I don't think we should couple the docker-helper or the release-helper to the `Makefile`, that's even tighter than this coupling with `setuptools_scm`.
I would propose to keep track of this and tackle this in the coming days (in the next iteration), after some discussion and playing around with possible ways to circumvent this.

Reply:
FYI: This is being tackled in https://github.com/localstack/localstack/pull/11441 by @silv-io! 🚀